extension/rikaichan.ts

```diff
@@ -263,7 +263,6 @@ class RcxMain {
 
     const m = this.showMode;
     let e: DictEntryData | null = null;
-
```

Comment:
Looks like there was a whitespace change left in.

We should definitely talk about dev environments since some flows will make this more obvious.